include/inviwo/core/util/filesystem.h

```diff
@@ -371,16 +379,25 @@ IVW_CORE_API bool isAbsolutePath(const std::string& path);
  * @return true if queryPath and refPath are located on the same drive (on Windows), always true on
  * all other systems
  */
-IVW_CORE_API bool sameDrive(const std::string& refPath, const std::string& queryPath);
+[[deprecated]] IVW_CORE_API bool sameDrive(const std::filesystem::path& refPath,
+                                           const std::filesystem::path& queryPath);
 
 /**
  * \brief clean up path by replacing backslashes with forward slash and removing surrounding quotes
  *
  * @param path given path to be cleaned up
  * @return non-quoted path containing no backslashes as directory separators
  */
-IVW_CORE_API std::string cleanupPath(std::string_view path);
+[[deprecated]] IVW_CORE_API std::filesystem::path cleanupPath(std::string_view path);
```

Comment:
Is this replaced by stripIdentifier? ```suggestion
[[deprecated]] IVW_CORE_API std::filesystem::path cleanupPath(std::string_view path);
```
```suggestion
[[deprecated("Use util::stripIdentifier() instead.")]] IVW_CORE_API std::filesystem::path cleanupPath(std::string_view path);
```